### DISCUSSION TITLE
WaterfallChart

### DISCUSSION THREAD

**Discussion by Maetes** [mantinedev/mantine#6224] [Feature requests]
Hi,

I would like to propose the addition of a WaterfallChart feature to the project. Given that this chart type is closely related to the Stacked Bar Chart, it's feasible to implement based on the existing Bar Chart functionality. 

Let me know if there is interest in officially integrating this feature, or if it is preferred doing this by myself. For the former, I'm happy to prepare a pullrequest.

cheers

**Comment by rtivital** [comment]
You are welcome to submit a PR with this feature. It should be a part of BarChart component with `type="waterfall"` if possible.

**Reply by Maetes** [reply]
Thank you. Happy to hear that. I'll do so.
